ena-submission/Snakefile

```diff
@@ -15,6 +15,9 @@ for key, value in defaults.items():
 
 LOG_LEVEL = config.get("log_level", "INFO")
 SUBMIT_TO_ENA_PROD = config.get("submit_to_ena_prod", False)
+if config.get("backend_url", "") not in config.get("allowed_submission_hosts", []):
```

Comment:
My expectation would have been that this wouldn't be the external URL but I may be wrong

Reply:
On this preview the backend_url that is passed here is https://backend-update-config.loculus.org

Reply:
So I think this is correct?

Reply:
Ah interesting; @corneliusroemer this potentially complicates your password plans

Reply:
So basically we should probably change the backend URL to the internal version eventually. I guess the concern is we might do that and still rely on this check which would no longer work

Reply:
A bit tedious but we could still make that change by first setting this list to include the external and internal url, changing the urls to internal and then removing the external url.

Reply:
Well I think the internal URL will be the same across deployments. But yes, you're right. We wouldn't accidentally make the check pass – I was thinking of my approach of banning "loculus.org" not your allowList approach, so this is aace

Reply:
ah wupps - if they are all the same we might have to add a better check! But yeah we can do that in future - thanks for all the help and feedback!